cmd/capdctl/main.go

```diff
@@ -143,6 +158,41 @@ spec:
   providerSpec: {}`, name, namespace)
 }
 
+func machineDeploymentYAML(opts *machineDeyploymentOptions) string {
+	deployment := v1alpha1.MachineDeployment{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "MachineDeployment",
+			APIVersion: "cluster.k8s.io/v1alpha1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      *opts.name,
+			Namespace: *opts.namespace,
+			Labels: map[string]string{
+				"cluster.k8s.io/cluster-name": *opts.clusterName,
+				"set":                         "node",
+			},
+		},
+		Spec: v1alpha1.MachineDeploymentSpec{
+			Replicas: opts.replicas,
+			Template: v1alpha1.MachineTemplateSpec{
+				Spec: v1alpha1.MachineSpec{
+					Versions: v1alpha1.MachineVersionInfo{
+						ControlPlane: *opts.controlPlaneVersion,
```

Comment:
we don't ever want to set the control plane version since this machine deployment will only manage worker nodes